CHANGES

```diff
@@ -2,6 +2,8 @@ arpack-ng - 3.6.3
 
 [ Franck Houssen ]
  * Add Fortran common initialization (block data).
+ * [BUG FIX] ICB: missing workev for *[ds]neupd (real+not-sym) => API/ABI change for *[ds]neupd_c.
```

Comment:
Could you please separate the two changes?

Reply:
Not sure to understand. This comment is related to #155 (the PR this one is based on). Do you want I split #155 in 2 ?